servicecontrol/capacity-and-planning.md

```diff
@@ -61,4 +61,31 @@ The transports supported by ServiceControl out-of-the-box (i.e. MSMQ, RabbitMQ,
 
 Azure Queues and Service Bus throughput varies significantly based on deployment options and multiple related variables inherent to cloud deployment scenarios.
 
-It is recommended that you plan and perform realistic throughput tests on ServiceControl using the transport of your choice and deployment options that are as close as possible to your planned production deployment. For additional questions or information please [contact Particular Software](http://particular.net/contactus).
\ No newline at end of file
+It is recommended that you plan and perform realistic throughput tests on ServiceControl using the transport of your choice and deployment options that are as close as possible to your planned production deployment. For additional questions or information please [contact Particular Software](http://particular.net/contactus).
+
+#### ServiceControl in Practice
+
+Often customers don't need all the features available in ServiceControl in one single environment. Its a tool that can accomodate many situations and sometimes some features get used in situations where their value proposition is not particularly strong.
```

Comment:
Perhaps mention that each feature comes with a cost and that cost doesn't need to be paid in every environment.